### PR TITLE
Add translation support.

### LINKED ISSUE #18: UI internationalization
Add a Norwegian UI. Just Norwegian, no other languages. JK, support as many languages as we can.

### PR DESCRIPTION
**Pull request by xslendix**
Closes #18.

### CODE REVIEW

File: dist.py [hunk 1]
```diff
@@ -51,6 +53,20 @@
...
 os.system(f"{venv_path}pyside6-uic ui/confirm.ui -o src/nexus/ui/ConfirmDialog.py")
 
+print("Generating TS template...")
+os.system(f"{venv_path}pyside6-lupdate " +
+          ' '.join(["ui/main.ui",
+                    "ui/banlist.ui",
+                    "ui/banword.ui",
+                    "ui/confirm.ui",
+                    "src/nexus/GUI.py"]) +
+          " -ts translations/i18n_en.ts")
```
Comment (Raymo111) [inline dist.py:63]: Let's update this dynamically for any and all future ui files

File: src/nexus/GUI.py [hunk 2]
```diff
@@ -84,12 +101,12 @@ def stop_logging(self):
 
     def start_stop(self):
         """Controller for start/stop logging button"""
-        if self.start_stop_button.text() == "Start logging":
+        if self.start_stop_button.text() == self.translator.translate("GUI", "Start logging"):
```
Comment (Raymo111) [inline src/nexus/GUI.py:104]: Can we use `self.tr` instead of `self.translator.translate` for all of these?

**Review by Raymo111 — CHANGES_REQUESTED** [review]
Looks great, almost there!

File: src/nexus/GUI.py (hunk 2 above)
Comment (xslendix) [reply src/nexus/GUI.py:104]: We do not have `self.tr` here, `self.translator` needs to be used.

Comment (Raymo111) [reply src/nexus/GUI.py:104]: Couldn't we bind the function? 🙂

**Review by Raymo111 — APPROVED** [review]

**Pull request merged**